eclipse-theia/theia
# Improve localization for preferences and monaco

#### What it does

Closes https://github.com/eclipse-theia/theia/issues/12372

Fixes a few minor discrepencies between how VSCode localizes preferences and how Theia does it. Also adds a few translation calls where they were missing.

The `test.svg` file deletion is a leftover fix from https://github.com/eclipse-theia/theia/pull/9864. The view has been using `codicon('beaker')` for quite some time now.

#### How to test

Install a localization pack and confirm that everything is localized as expected.

#### Review checklist


#### Reminder for reviewers

- As a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)
